parlai/core/dialog_teacher.py

```diff
@@ -240,6 +242,7 @@ def _load(self, data_loader):
                                 new_entry.append(None)
                             if len(entry) > 4 and entry[4] is not None:
                                 new_entry.append(sys.intern(entry[4]))
+                                self.image_loader = ImageLoader(opt) 
```

Comment:
this will create a new image loader on every example with an image--not good. since the image initializer is lazy anyways, let's move this to `__init__`